prog/dftb+/lib_dftb/scc.F90

```diff
@@ -387,6 +390,8 @@ subroutine Scc_initialize(this, env, inp)
       this%h5Correction = inp%h5Correction
     end if
 
+    this%hasExternalShifts = .false.
```

Comment:
Shouldn't we get this information from the input? Setting this here to a fixed value and flipping it later without a way to reset it is somewhat hacky.